amida-tech/indaba-client
# Styling/inba 219 font fixes

#### What's this PR do?
UI styling fixes for the PM dashboard page. 
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-219
#### How should this be manually tested?
Use the style sheet to compare styles. I will add styles to the sheet throughout the process. Each font should be an extension from the font pages. Each input and table should be a global declaration.
#### Any background context you want to provide?
The global $border-color does not work on the search-input which is why youll see the hex value for it in the scss file.
![style guide desktop](https://user-images.githubusercontent.com/16657726/33562648-c1ce7f18-d8e3-11e7-83a0-0637cf42ebe0.jpg)

#### Screenshots (if appropriate):
